app/com/arpnetworking/metrics/portal/hosts/impl/ForemanHostProvider.java

```diff
@@ -44,17 +45,17 @@
      * @param configuration Play configuration.
      */
     @Inject
-    public ForemanHostProvider(final HostRepository hostRepository, final Configuration configuration) {
+    public ForemanHostProvider(final HostRepository hostRepository, @Assisted final Configuration configuration) {
         _hostRepository = hostRepository;
         getContext().system().scheduler().schedule(
-                ConfigurationHelper.getFiniteDuration(configuration, "hostProvider.initialDelay"),
-                ConfigurationHelper.getFiniteDuration(configuration, "hostProvider.interval"),
+                ConfigurationHelper.getFiniteDuration(configuration, "initialDelay"),
+                ConfigurationHelper.getFiniteDuration(configuration, "interval"),
                 getSelf(),
                 TICK,
                 getContext().dispatcher(),
                 getSelf());
         _client = new ForemanClient.Builder()
-                .setBaseUrl(URI.create(configuration.getString("hostProvider.foreman.baseUrl")))
+                .setBaseUrl(URI.create(configuration.getString("foreman.baseUrl")))
```

Comment:
foreman.baseUrl is a root-level key of this host provider.  Technically, it's actually foreman as the root-level key and baseUrl is a subkey of foreman.